controllers/apply/awards-for-all/fields.js

```diff
@@ -1019,6 +1046,65 @@ module.exports = function fieldsFor({ locale, data = {} }) {
         };
     }
 
+    function fieldProjectTotalCosts() {
+        const budgetTotal = sumBy(
+            getOr([], 'projectBudget')(data),
+            item => parseInt(item.cost, 10) || 0
+        );
+
+        return {
+            name: 'projectTotalCosts',
+            label: localise({
+                en: 'Tell us the total cost of your project',
+                cy: '(WELSH) Tell us the total cost of your project'
+            }),
+            explanation: localise({
+                en: oneLine`<p>
+                    This is the cost of everything related to your project,
+                    even things you aren't asking us to fund.
+                </p>
+                <p>
+                    For example, if you are asking us for £8,000 and you are
+                    getting £10,000 from another funder to cover additional costs,
+                    then your total project cost is £18,000.
+                    If you are asking us for £8,000 and there are no other costs
+                    then your total project cost is £8,000.
+                </p>`,
+                cy: ``
+            }),
+            type: 'currency',
+            isRequired: true,
+            schema: Joi.friendlyNumber()
```

Comment:
`friendlyNumber` is a new low level extension which is solely responsible for allowing `,` and spaces in numbers and coercing it to a number

Reply:
Does it also allow spaces in numbers ?

Reply:
I have also noticed, if you pass a value like `21ab5c6.00` to friendly-number, we assume its correct and convert it to `2156` whilst not throwing any validation error. Is this intended ?